README.md

```diff
@@ -11,3 +11,38 @@ TODO: Add detailed app description and demo
 ![architecture diagram](images/architecture-diagram.png)
 [Image Link](https://excalidraw.com/#json=1c_DrTFZCSGprCvJNBYHn,B0sXrISCY8YdKgiTqy9xDA)
 _revised 9/4/2024_
+
+The structure of the event database is as follows:
+ ```json
+ {
+  "Event": {
+    "Name": "New Years Eve Party",
+    "Photo": "https://stackoverflow.com/questions/46585330/firestore-security-rules-for-public-and-private-fields",
+    "Description": "Get Ready for NYE 2025, dress classy and dance all night",
+    "Category": "Party",
+    "Start": {
+      "Date": "31.12.2024",
```

Comment:
nit: you might want to check to see what the string format of `LocalDate` and `LocalTime` is.

Reply:
Checked and modified